src/components/Todo/TodoComponent.tsx

```diff
@@ -33,6 +35,12 @@ const TodoComponent = ({
     setTodoValue(initialValue);
   }, [initialValue]);
 
+  useEffect(() => {
+    if (clearTrigger) {
+      setTodoValue(''); // Clear the text field when clearTrigger changes
+    }
+  }, [clearTrigger]);
```

Comment:
Can be refactored, right now, there is a bug where if you add 2 todos, it will not reset after the first one. I think the clearing of the value should be done on the parent component; in line to this, I think the value in line 31 should be a prop and should driven by the parent component.